deploy/longhorn-images.txt

```diff
@@ -1,8 +1,8 @@
-longhornio/csi-attacher:v2.2.1-lh1
-longhornio/csi-node-driver-registrar:v1.2.0-lh1
-longhornio/csi-provisioner:v1.6.0-lh1
-longhornio/csi-resizer:v0.5.1-lh1
-longhornio/csi-snapshotter:v2.1.1-lh1
+longhornio/csi-attacher:v3.2.1
```

Comment:
It seems like use the cloned image. Please also make the change to 1.2.x and master.

Reply:
ref: https://github.com/longhorn/longhorn/issues/3056 cc @meldafrawi